hello/src/hello.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "hello.h"
+
+char *
+hello(char *str) {
+  int size = strlen(str) + 10;
+  char *message = (char *) malloc(size);
+
+  if (message == NULL) {
+    exit(EXIT_FAILURE);
```

Comment:
I understood you said. 👍
but should I return what value? I'm wondering if I should `return EXIT_FAILURE` is not char function, so umm.